landingai/predict/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .edge import EdgePredictor  # noqa: F401
+from .cloud import Predictor  # noqa: F401
+from .ocr import OcrPredictor  # noqa: F401
+from .snowflake import SnowflakeNativeAppPredictor  # noqa: F401
```

Comment:
This keeps the imports structure as they are today; so, no breaing changes for end users.